scripts/generate-and-apply-external-dns-yaml.sh

```diff
@@ -102,3 +102,7 @@ tanzu package install external-dns \
     --namespace tanzu-kapp \
     --values-file generated/$CLUSTER_NAME/external-dns/external-dns-data-values.yaml \
     --poll-timeout 10m0s
+
+# Apply overly for metrics
```

Comment:
nitpick: replace overly with overlay

Reply:
Good question.  I've been testing collection using Tanzu Observability using the prometheus endpoints.  I've created a custom dashboard for common Tanzu Standard Packages: CertManager, Harbor and External Dns.  But I have not tested with the Tanzu Standard Prometheus/Grafana packages.

Reply:
Fixed overlay spelling error.